tests/app/main/test_frameworks.py

```diff
@@ -1613,55 +1614,81 @@ def teardown_method(self, method):
         self.data_api_client_patch.stop()
         super().teardown_method(method)
 
-    expected = (
-        'Your application will be submitted at 5pm&nbsp;BST,&nbsp;Tuesday&nbsp;6&nbsp;October&nbsp;2015. <br> '
-        'You can edit your declaration and services at any time before the deadline.'
-    )
-
-    def test_confidence_banner_on_page(self, _):
-        """Test confidence banner appears on page happy path."""
+    def test_success_banner_on_page_for_open_framework(self, _):
         self.data_api_client.find_draft_services_iter.return_value = [
             {'serviceName': 'A service', 'status': 'submitted', 'lotSlug': 'foo'}
         ]
-        self.data_api_client.get_supplier_framework_info.return_value = self.supplier_framework(status='complete')
-        self.data_api_client.get_supplier.return_value = SupplierStub().single_result_response()
+        self.data_api_client.get_supplier_framework_info.return_value = self.supplier_framework(
+            status='complete',
+            application_company_details_confirmed=True,
+        )
+        self.data_api_client.get_supplier.return_value = SupplierStub(
+            company_details_confirmed=True).single_result_response()
 
         self.login()
         res = self.client.get("/suppliers/frameworks/g-cloud-8")
         assert res.status_code == 200
-        assert self.expected in str(res.data)
+
+        document = html.fromstring(res.get_data(as_text=True))
+
+        alert_banner = document.xpath('//div[@class="dm-alert dm-alert--success"]')
+        assert len(alert_banner) == 1
+        assert alert_banner[0].xpath(
+            "//h2[contains(normalize-space(string()), $t)]",
+            t="Your application is complete, and will be automatically submitted.",
```

Comment:
Suggest amend: Your application is complete and will be submitted automatically.